packages/tracking/src/test/trackEvent.ts

```diff
@@ -34,23 +34,23 @@ describe('trackEvent', () => {
     config.trackingAllowed = true;
     config.trackingEnabled = true;
 
-    gtag.mockImplementationOnce((type, eventName, eventData) => {
-      eventData.event_callback();
+    jest.mocked(gtag).mockImplementationOnce((_type, _eventName, eventData) => {
+      (eventData as ControlParams).event_callback?.();
     });
 
-    await trackEvent('name', { foo: 'abc', bar: 'def', baz: 'ghi' });
+    await trackEvent('name', { name: 'abc', value: 'def', event_label: 'ghi' });
```

Comment:
Is there a way to still test with unexpected props? In case we want to test that specifically.

Reply:
Yeah, by ignoring TypeScript errors.